restapi/stories.go

```diff
@@ -190,9 +190,26 @@ func (api api) CreateStoriesFormData(w http.ResponseWriter, r *http.Request) ren
 	if err := api.database.Create(&story).Error; err != nil {
 		return rest.ErrInternal(api.logger, err)
 	}
+	fmt.Println(r.FormValue("bio"))
+	names := r.Form["tags"]
+	err = api.AddTags(names, story)
+	if err != nil {
+		return rest.ErrInvalidRequest(api.logger, "Error Adding Tags", err)
+	}
 	return rest.MsgStatusOK("Story Added Successfully")
 }
 
+func (api api) AddTags(names []string, story models.Story) error {
```

Comment:
consider making this a helper method instead of a method on api

Reply:
isnt it better to have it as a method on api, because otherwise i would have to pass in db in parameters of the function??

Reply:
Oh i didn't notice you use db, then its fine to keep it in api